megatron/core/inference/engines/dynamic_engine.py

```diff
@@ -136,6 +137,7 @@ def __init__(
         track_paused_request_events: bool = False,
         enable_chunked_prefill: bool = True,
         inference_logging_step_interval: int = 0,
+        process_group_collection: Optional[ProcessGroupCollection] = None,
```

Comment:
Q: Other places all used `pg_collection`, any reason it needs to be more explicit here `process_group_collection`? If not, suggest to be consistent and use `pg_collection`